osh/hub/settings.py

```diff
@@ -188,28 +192,15 @@
     pass
 
 
-def _get_secret(name):
-    try:
-        with open(os.path.join(SECRETS_DIR, name)) as f:
-            return f.read().strip()
-    except OSError:
-        return None
-
+###############################################################################
+# Secrets
+###############################################################################
 
-BZ_API_KEY = _get_secret('bugzilla_secret')
-JIRA_API_KEY = _get_secret('jira_secret')
+from osh.hub.other.settings_util import get_secret  # noqa: E402
 
-# Denote whether the access to user list/detail view is restricted
-# Possible values:
-# "" (empty string) = Anonymous access (default)
-# "authenticated" = Authenticated users
-# "staff" = Staff (admin) users only
-USERS_ACL_PERMISSION = "staff"
+# Issue trackers
+BZ_API_KEY = get_secret('bugzilla_secret', SECRETS_DIR)
+JIRA_API_KEY = get_secret('jira_secret', SECRETS_DIR)
 
-# read the real SECRET_KEY from SECRET_KEY_FILE if availble
-try:
-    with open(SECRET_KEY_FILE) as f:
-        key = f.readline()
-        SECRET_KEY = key.strip()
-except OSError:
-    pass
+# Secret key ((will be overridden by the content of SECRET_KEY_FILE if available)
```

Comment:
`((` -> `(`

Reply:
Addressed in ff868222d7d675213c74a4dfffabd63d0a8d4f84.